payments/utils.py

```diff
@@ -1,17 +1,14 @@
 import calendar
 from datetime import date, timedelta
-from decimal import Decimal, ROUND_HALF_UP
+from decimal import Decimal
 from functools import wraps
 from typing import Callable
 
 from dateutil.relativedelta import relativedelta
 from dateutil.rrule import MONTHLY, rrule
 
 from leases.utils import calculate_season_end_date, calculate_season_start_date
-
-
-def round_price(price):
-    return price.quantize(Decimal("0.01"), rounding=ROUND_HALF_UP)
+from utils.numbers import rounded as util_rounded
```

Comment:
should this be `as rounded_decimal` for consistency. and also we are in the `utils.py` file and there is `rounded` decorator function here

Reply:
Makes more sense 👍